plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -2678,10 +2678,12 @@ read_only::get_consensus_parameters_results
 read_only::get_consensus_parameters(const get_consensus_parameters_params&, const fc::time_point& ) const {
    get_consensus_parameters_results results;
 
-   if (db.is_builtin_activated(builtin_protocol_feature_t::action_return_value))
-      to_variant(db.get_global_properties().configuration,        results.chain_config); //chain_config_v1
+   if (db.is_builtin_activated(builtin_protocol_feature_t::sync_call))
+      to_variant(db.get_global_properties().configuration, results.chain_config); //chain_config_v2
+   else if (db.is_builtin_activated(builtin_protocol_feature_t::action_return_value))
+      to_variant(db.get_global_properties().configuration.base(), results.chain_config); //chain_config_v1
    else
-      to_variant(db.get_global_properties().configuration.base(), results.chain_config); //chain_config_v0
+      to_variant(db.get_global_properties().configuration.v0(), results.chain_config); //chain_config_v0
```

Comment:
Alternatively `base().base()` if you don't want to add the new `v0()`. I don't have a strong preference either way